### PR TITLE
[new release] ocamlformat, ocamlformat-rpc and ocamlformat-rpc-lib (0.18.0)

### PR DESCRIPTION
**Pull request by gpetiot**
Auto-formatter for OCaml code

- Project page: <a href="https://github.com/ocaml-ppx/ocamlformat">https://github.com/ocaml-ppx/ocamlformat</a>

#### Bug fixes

  + Fix extraneous parenthesis after `let open` with `closing-on-separate-line` (#1612, @Julow)

  + Add missing break between polytype quantification and arrow-type body (#1615, @gpetiot)

#### Changes

  + Use dune instrumentation backend for `bisect_ppx` (#1550, @tmattio)

  + Format objects and classes consistently with structure and signature items (#1569, @bikallem)

#### New features

  + Expose a RPC interface through a new binary `ocamlformat-rpc` and a new library `ocamlformat-rpc-lib` (#1586, @gpetiot, @voodoos)

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
The ocamlformat tests and the new ocamlformat-rpc package fail. Could you have a look at the failures?

**Comment by kit-ty-kate** [comment]
there are still failures on ocamlformat-rpc and ocamlformat-rpc-lib's tests

**Comment by kit-ty-kate** [comment]
> I've reverted your change for now as I think it shouldn't be the issue, I might put it back later if I'm wrong!

dune-release somehow didn't change my change, it works well with but I just ""reverted"" it so let's see if this is enough.

File: packages/ocamlformat-rpc/ocamlformat-rpc.0.18.0/opam [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+maintainer: ["OCamlFormat Team <ocamlformat-dev@lists.ocaml.org>"]
+authors: ["Josh Berdine <jjb@fb.com>"]
+license: "MIT"
+homepage: "https://github.com/ocaml-ppx/ocamlformat"
+bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08" & < "4.13"}
+  "ocaml-version"
+  "ocamlformat-rpc-lib"
```
Comment (kit-ty-kate) [inline packages/ocamlformat-rpc/ocamlformat-rpc.0.18.0/opam:14]: is `ocamlformat-rpc-lib` always going to be released at the same time as `ocamlformat-rpc`? If yes, could `{= version}` be added?

Comment (gpetiot) [reply packages/ocamlformat-rpc/ocamlformat-rpc.0.18.0/opam:14]: I guess they will be released at the same time but it shouldn't be an issue to have ocamlformat-rpc-lib 0.19.0 with ocamlformat-rpc 0.18.0 (or vice versa) in the future

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**